b2b-sample/guardio-insurance-sample-application/components/settingsComponents/addUserComponent.js

```diff
@@ -75,12 +82,13 @@ export default function AddUserComponent(props) {
 
     const validate = values => {
         const errors = {}
-        errors = nameValidate(values.name, errors);
+        errors = firstNameValidate(values.firstName, errors);
+        errors = familyNameValidate(values.familyName, errors);
         errors = emailValidate(values.email, errors);
         errors = usernameValidate(values.username, errors)
         errors = passwordValidate(values.password, errors);
         errors = repasswordValidate(values.repassword, errors);
-
+        console.log(errors);
```

Comment:
Shall we remove console logs?

Reply:
Check other places too.

Reply:
These have been removed with later PR's, as there are no console.log 's in unnecessary places.